test/Passes/pass-parallel-tile.mlir

```diff
@@ -0,0 +1,47 @@
+//RUN: tpp-opt %s  --scf-parallel-loop-tiling-pass -split-input-file | FileCheck %s
+
+  func.func @entry() {
```

Comment:
You really need more tests. At least one GEMM and one MLP layer checking the right loops are being created.

I'd also create smaller tests with different pass tile options and make sure the right split is done.

Reply:
I have added gemm and mlp test cases with different grid sizes generated from mlir-gen to ensure that the splits are correct.